pkg/crc/telemetry/telemetry.go

```diff
@@ -84,3 +86,12 @@ const (
 	CreationStartType       StartType = "creation"
 	StartStartType          StartType = "start"
 )
+
+func SetError(err error) string {
+	// Mask username if present in the error string
+	user, err1 := user.Current()
+	if err1 != nil {
+		return err1.Error()
+	}
+	return strings.ReplaceAll(err.Error(), user.Username, "XXXXX")
```

Comment:
`XXXX` would be more readable as `$USER` or something like this. I'd go with a (precompiled) regexp to be able to match `'\b' + user.Username + '\b'\` (only match full words, to avoid some false positives)

Reply:
I'd suggest something confusing like `fluffy-cat` or `blind-mole`. Would be the best thing to stumble on in Woopra :)

Reply:
@cfergeau regex is bit expensive than strings replace, when you say `false positive` you mean if the same username is part of the any error substring and if that occur we can always improve.

Reply:
regex matching, especially precompiled is not going to be more expensive than the network traffic which is triggered shortly after this masking. We are already going to get some false positives when generic usernames are used (`ubuntu`). If we only match full words, this will at least avoid false positives on short usernames (made up one, but if someone picks `unn`, it's better if we don't match it in `Running`).